multi-node/aws/pkg/config/config.go

```diff
@@ -136,6 +136,8 @@ type Cluster struct {
 	KMSKeyARN                string            `yaml:"kmsKeyArn,omitempty"`
 	CreateRecordSet          bool              `yaml:"createRecordSet,omitempty"`
 	RecordSetTTL             int               `yaml:"recordSetTTL,omitempty"`
+	TLSCADurationDays        int               `yaml:"tlsCAValidDurationDays,omitempty"`
```

Comment:
@pbx0 you should set default values [here](https://github.com/pbx0/coreos-kubernetes/blob/3ff4b7e2bbc80dfdeaa50e4bfc16ec7e0a2fadb2/multi-node/aws/pkg/config/config.go#L32) for the durations. That will fix the unit tests.